BREAKING_CHANGES.md

```diff
@@ -3,4 +3,5 @@
 ## 2.0.0
 
 * Added support .net 7
-* Dropped support for .net 6
\ No newline at end of file
+* Dropped support for .net 6
+* Refactored KeyName to Key and KeyValue to Value
```

Comment:
We could say in the IBiometryService here.

Reply:
Refactored IBiometryService KeyName to Key and KeyValue to Value?

Reply:
Yes it would be better